core-v1/src/main/scala/algebra/Extract.scala

```diff
@@ -0,0 +1,26 @@
+package com.rallyhealth.vapors.v1
+
+package algebra
+
+import data.Justified
+
+import cats.Id
+
+// TODO: Replace with cats Monoid or ZIO Prelude?
+trait Extract[F[_]] {
+
+  def extract[A](fa: F[A]): A
+}
```

Comment:
There is a typeclass for `Extract` in alleycats... not sure if I want to pull in all of alleycats though...